src/components/OverlayControls/OverlayControls.tsx

```diff
@@ -294,13 +331,20 @@ class OverlayControls extends React.Component<OverlayControlsProps> {
                       // @ts-expect-error
                       text: item.title || i18n(item.id),
                       iconStart: item.icon,
+                      // c9c3d089-fe7f-4ae3-8557-892939db8874
```

Comment:
It's better use more informative comments instead

Reply:
replaced it with links